viewing_party/party.py

```diff
@@ -1,23 +1,116 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie_dict = {}

```

Comment:
Great job with this method! Couple of small things to note.

You could put "return None" into it's own guard clause with (on line 4 so it's the first line of code that executes in this method):
```python
if not title or not genre or not rating:
      return None
```
and after that, proceed to create your movie dict

This way, when another developer comes along, they can immediately recognize the guard clause pattern. The way your code is currently structured, I can tell at some point this method will return None, but it's not as immediately clear.

Also a note about guard clauses - The Guard Clause pattern helps in these cases by “inverting” that way of thinking and rapidly returning false conditions and keeping the important code blocks (usually the longest) in the lowest indentation level possible. 

You can also create your dictionary with literal syntax and return it without first creating an empty dict on line 4:

return {
        "title": title,
        "genre": genre,
        "rating": rating        
    }


So your function would immediately have a guard clause that returns None and then the next line of code would return a dictionary with your movie data.